client/omnichannel/directory/ContactForm.js

```diff
@@ -190,7 +213,7 @@ export function ContactNewEdit({ id, data, reload, close }) {
 			<Field>
 				<Field.Label>{t('Email')}</Field.Label>
 				<Field.Row>
-					<TextInput error={emailError} flexGrow={1} value={email} onChange={handleEmail} addon={<Icon name='mail' size='x20'/>}/>
+					<TextInput onBlur={checkEmailExists} error={emailError} flexGrow={1} value={email} onChange={handleEmail} addon={<Icon name='mail' size='x20'/>}/>
```

Comment:
When a set an email and remove it, I get this:

<img width="643" alt="Screen Shot 2021-01-12 at 12 20 04" src="https://user-images.githubusercontent.com/59577424/104334162-cb8edb80-54d0-11eb-86de-ba446543c988.png">